### PR TITLE
Extension: Refresh the visual UI

### PR DESCRIPTION
**Pull request by jasmussen**
A small PR to update the visual styles of the extension:

Screenshot 2024-11-11 at 16 22 30
Screenshot 2024-11-11 at 16 22 42

### CODE REVIEW

**Comment by psrpinto** [comment]
It's looking great, well done @jasmussen. Thank you!

**Comment by ashfame** [comment]
@jasmussen Thanks for the work in this PR! Can I make a few suggestions?

- Playground preview pane is now completely empty when the experience begins. Would love to have a playground logo or something there.
- Tabs view look weird when viewing the Preview tab with the admin bar kind of hiding the bottom of it being emphasized. Perhaps we can use a different type of emphasis, like an oval border or something?
- `Clear` & `Cancel` button when selecting fields shouldn't be a primary, but a ghost/outline version?

File: src/ui/preview/PlaceholderPreview.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export function PlaceholderPreview() {
 	return (
 		<>
-			<span>Placeholder Preview</span>
+			<span></span>
```
Comment (ashfame) [inline src/ui/preview/PlaceholderPreview.tsx]:
```suggestion
			<img
				src="https://wordpress.org/playground/files/2024/06/playground-ready.png"
				alt="playground-image"
				className="playground-preview-splash"
			/>
```

File: src/ui/app.css [hunk 2]
```diff
@@ -58,55 +172,39 @@ ol {
...
-}
-
 .preview-tabs-tab label {
-	padding: 10px;
+	padding: var(--spacing-sm) var(--spacing-lg);
 }
 
 .preview-tabs-tab input {
 	display: none;
 }
```
Comment (ashfame) [inline src/ui/app.css:226]:
```suggestion

.playground-preview-splash {
	width: 80%;
	margin: auto;
}
```

**Review by ashfame — COMMENTED** [review]
Made a code suggestion in my review to add a splash screen, looks like this on my local:

Screenshot 2024-11-12 at 01 45 02

**Comment by jasmussen** [comment]
Hey, excellent notes, thanks! For now I simply removed everything from the preview window, instead of having placeholder text, though I agree that a tiny logo could work there. Though I would not use that Playground asset, since it's both work in progress and an illustration rather than a logo. We could add a tiny gray WordPress logo, centered there. This would match the extension icon itself, which I also had hopes of updating. 

I've limited time today, to further this work, so feel free to merge this as is, or update the PR to meet expectations. Thank you!

**Comment by ashfame** [comment]
| Before | After |
|--------|--------|
| Screenshot 2024-11-12 at 13 53 49 | Screenshot 2024-11-12 at 13 53 13 |

| Before | After |
|--------|--------|
| `blank` | Screenshot 2024-11-12 at 14 07 38 |

**Comment by jasmussen** [comment]
Extension icon looks way better! For the preview icon, can you make it half the size, and much lighter? Perhaps just `opacity: 0.3;`?

**Comment by ashfame** [comment]
Looks great with 0.3 opacity:

Screenshot 2024-11-12 at 14 31 52

**Comment by ashfame** [comment]
Forgot size, here it is:

Screenshot 2024-11-12 at 14 33 52

**Comment by jasmussen** [comment]
Looks great, ship it!

**Comment by ashfame** [comment]
Wanted to reduce the prominence of non-primary buttons & improve that screen a bit:

| Before | After |
|--------|--------|
| Screenshot 2024-11-12 at 15 13 02 | Screenshot 2024-11-12 at 15 09 02 |

@jasmussen If you think, this is ok, that's the last of it for now before we ship it :)

**Review by psrpinto — APPROVED** [review]
This is looking great, thank you both for working on this!

**Comment by jasmussen** [comment]
Nice, yeah the selectors I'd love to take a stab at too. For now that text-only button style for clear and cancel looks good. When I get a momen to return, I'd like to style the fieldset lightly. Finally, we should make sure if there's a primary button and a text only button together, the primary button should always be first. But no blocker to merging! Thanks for letting me contribute, and for the help.

**Comment by ashfame** [comment]
@jasmussen Good tip! I made another variant as per that, which I will ship for now and afterwards we can do it properly.

screenshot_2024-11-12_at_15 29 39_720

Thank you for your assistance! 🚀

**Pull request merged**